xbmc/playlists/PlayListB4S.cpp

```diff
@@ -50,26 +50,33 @@ CPlayListB4S::~CPlayListB4S(void) = default;
 
 bool CPlayListB4S::LoadData(std::istream& stream)
 {
-  CXBMCTinyXML xmlDoc;
+  CXBMCTinyXML2 xmlDoc;
 
-  stream >> xmlDoc;
+  std::string b4sstream(std::istreambuf_iterator<char>(stream), {});
```

Comment:
For future, camelcase (`b4sStream`) is intended to improve readability. But such a small change, no need to update this PR.

Reply:
fixed, for b4s as well as asx and wpl